erpnext_documentation/www/docs/user/manual/en/manufacturing/bill-of-materials.md

```diff
@@ -106,7 +106,25 @@ The BOM can be linked to a [Project](/docs/user/manual/en/projects) to track pro
 
 The BOM can also be shown in the [Website](/docs/user/manual/en/website) for Open-source hardware products. Open-source hardware is similar to open-source where the product specifications are listed publicly.
 
-### 3.4 After Submitting
+### 3.4 BOM Template
+
+<img class="screenshot" alt="BOM Template" src="{{docs_base_url}}/assets/img/manufacturing/bom-template.png">
+
+For template item (against which user creates the variant items), user can make the BOM and that BOM can be used as a default BOM while making the Work Order against the respective template's variant item. User can also add the template item as raw materials in the template item. While making Work Order from the BOM, system gives provision to select the variant item against the Template Item, for more details check following screenshot.
+
+<img class="screenshot" alt="Variant Selection" src="{{docs_base_url}}/assets/img/manufacturing/variant-selection-against-template.png">
+
+The user can also make the BOM for the variant item using the template BOM. To make the variant BOM:
```

Comment:
recheck these steps. is there a BOM template doctype? how can the user access BOM template?